Bottom: 8e577561c1b8b1b1ba305f3877033e2fcf6d3f19
Top:    b457a4715ae5bb794c2973b4864d67507f24055f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 20:07:16 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
